objection/console/cli.py

```diff
@@ -255,18 +255,25 @@ def patchipa(source: str, gadget_version: str, codesign_signature: str, provisio
                                               'specified, the latest version will be used.'), default=None)
 @click.option('--pause', '-p', is_flag=True, help='Pause the patcher before rebuilding the APK.',
               show_default=True)
-@click.option('--skip-cleanup', '-k', is_flag=True,
+@click.option('--skip-cleanup', '-k', is_flag=True, default=True,
               help='Do not clean temporary files once finished.', show_default=True)
-@click.option('--enable-debug', '-d', is_flag=True,
+@click.option('--enable-debug', '-d', is_flag=True, default=True,
               help='Set the android:debuggable flag to true in the application manifest.', show_default=True)
 @click.option('--network-security-config', '-N', is_flag=True, default=False,
               help='Include a network_security_config.xml file allowing for user added CA\'s to be trusted on '
-                   'Android 7 and up. This option can not be used with the --skip-resources flag.')
+                   'Android 7 and up. This option can not be used with the --skip-resources flag.',
+              show_default=True)
 @click.option('--skip-resources', '-D', is_flag=True, default=False,
-              help='Skip resource decoding as part of the apktool processing.', show_default=False)
+              help='Skip resource decoding as part of the apktool processing.', show_default=True)
 @click.option('--target-class', '-t', help='The target class to patch.', default=None)
-@click.option('--use-aapt2', '-2', is_flag=True, default=False,
-              help='Use the aapt2 binary instead of aapt as part of the apktool processing.', show_default=False)
+@click.option('--use-aapt2', '-2', is_flag=True, default=True,
```

Comment:
Like `--skip-cleanup`, this flag now needs to specifically set the command to use 1.

Reply:
Indeed, I did the edit. I'd rather use `aapt2` by default (I cannot find any reason not to do so) so moved to an explicit `--use-aapt1` flag.